tools/check_docs.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# get all linked module docs for mkdocs.yml
+grep "modules/" mkdocs.yml | sed "s/ *- .*: *'//" | sed "s/'//" | sort > /tmp/doc
+
+# get all module and lua_module *.md files
+ls docs/modules/* docs/lua-modules/* | sed "sxdocs/xx" | sort > /tmp/files
```

Comment:
```suggestion
find docs/modules/*.md docs/lua-modules/*.md | sed "sxdocs/xx" | sort > /tmp/files
```

I ran this through shell-check and it complained about https://github.com/koalaman/shellcheck/wiki/SC2012

Reply:
Oh...and there's `lua-modules/README.md` 😉

Reply:
Saw that too, but it passes the test. If it is to be moved the test will have to be adapted